hudi-common/src/test/java/org/apache/hudi/common/functional/TestHoodieLogFormat.java

```diff
@@ -144,6 +144,7 @@ public void setUp() throws IOException, InterruptedException {
   @AfterEach
   public void tearDown() throws IOException {
     fs.delete(partitionPath, true);
+    fs.delete(new Path(basePath), true);
```

Comment:
this is not necessary as `tempDir` is annotated with `org.junit.jupiter.api.io.TempDir;` which will be cleaned up automatically